all-packages.nix

```diff
@@ -4,8 +4,16 @@
     gnunet-messenger-cli = callPackage ./pkgs/gnunet-messenger-cli {};
     liberaforms = callPackage ./pkgs/liberaforms {};
     liberaforms-env = callPackage ./pkgs/liberaforms/env.nix {};
+    kikit = callPackage ./pkgs/kikit {};
   };
 
-  callPackage = newScope self;
+  nixpkgs-candidates = {
+    pcbnew-transition = callPackage ./nixpkgs-candidates/pcbnew-transition {};
+    pybars3 = callPackage ./nixpkgs-candidates/pybars3 {};
+    pymeta3 = callPackage ./nixpkgs-candidates/pymeta3 {};
+    euclid3 = callPackage ./nixpkgs-candidates/euclid3 {};
```

Comment:
@mightyiam please add this pattern to documentation the `README`, including an note that upstreaming to Nixpkgs is encouraged.

Reply:
Added by https://github.com/ngi-nix/ngipkgs/pull/42